grammars/sql.cson

```diff
@@ -106,7 +106,7 @@
     'name': 'storage.modifier.sql'
   }
   {
-    'match': '\\b\\d+\\b'
+    'match': '\\b\\d+(\\.\\d+)?\\b'
```

Comment:
This seems to work for the most part: `\\b(\\d+(\\.(\\d+)?)?)|(\\.\\d+)\\b`
The only problem I can find is that the `3.` in `3.a` will be tokenized as a number when it shouldn't be.  Same with `3a`.